### PR TITLE
Implement Contact Full Screen

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes BLUI - 5119.

#### Changes proposed in this Pull Request:

- 

-
-

#### Screenshots / Screen Recording (if applicable)

-
Screenshot 2024-04-02 at 8 41 23 PM
Screenshot 2024-04-02 at 8 41 08 PM

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by ArshdeepSingh3-eaton** [comment]
add screenshot in contact documentation 
everything else looks good

**Comment by ektaghag-eaton** [comment]
> add screenshot in contact documentation everything else looks good

Updated. please review.

**Review by ArshdeepSingh3-eaton — APPROVED** [review]

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    const { navigate } = useAuthContext();
+    const theme = useExtendedTheme();
+    const { contactEmail = 'something@email.com', contactPhone = '1-800-123-4567' } = props;
+
+    const defaultEmailSupportContent = (): JSX.Element => (
+        <Text variant="bodyLarge">
+            {`${t('bluiAuth:CONTACT_SUPPORT.SUPPORT_MESSAGE')}`}
+            <Text
+                variant="labelLarge"
+                style={{ color: theme.colors.primary }}
```
Comment (surajeaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:30]: Please create a styles object using StyleSheet.create and use it here

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            {`.`}
+        </Text>
+    );
+
+    const defaultPhoneSupportContent = (): JSX.Element => (
+        <Text variant="bodyLarge">
+            {`${t('bluiAuth:CONTACT_SUPPORT.TECHNICAL_ASSISTANCE')}`}
+            <Text
+                variant="labelLarge"
+                style={{ color: theme.colors.primary }}
```
Comment (surajeaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:44]: Same comment as above

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import { useTranslation } from 'react-i18next';
+import { Text } from 'react-native-paper';
+import { useExtendedTheme } from '@brightlayer-ui/react-native-themes';
+import { Linking } from 'react-native';
+
+/**
+ * Component renders a screen with contact information for support with the application.
+ * Contact information is pulled from the context passed into the workflow.
+ *
+ * @param {ContactSupportScreenProps} props - Props of Create Account Screen
```
Comment (surajeaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:14]: This should be Props of Contact Support Screen

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+import { ContactSupportScreenProps } from './types';
+import { useAuthContext } from '../../contexts';
+import { useTranslation } from 'react-i18next';
+import { Text } from 'react-native-paper';
+import { useExtendedTheme } from '@brightlayer-ui/react-native-themes';
+import { Linking } from 'react-native';
+
+/**
+ * Component renders a screen with contact information for support with the application.
+ * Contact information is pulled from the context passed into the workflow.
```
Comment (surajeaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:12]: This sentence is not correct. In this screen, we are not pulling data from context.

**Pull request merged**